src/main/java/gregtech/api/recipes/RecipeBuilder.java

```diff
@@ -909,9 +911,6 @@ public R ignoreBuildAction(ResourceLocation buildActionName) {
             GTLog.logger.error("Recipe map {} does not contain build action {}!", recipeMap, buildActionName,
                     new Throwable());
             return (R) this;
-        } else if (ignoredBuildActions.containsKey(buildActionName)) {
-            GTLog.logger.error("Builder already ignores {}!", buildActionName, new Throwable());
-            return (R) this;
```

Comment:
You can keep the check to avoid doing extra work below, just remove the log message.